Sources/FluentKit/Query/QueryBuilder+Unique.swift

```diff
@@ -0,0 +1,16 @@
+extension QueryBuilder {
+    public func all<Result>(uniqueOn key: KeyPath<Model, Field<Result>>) -> EventLoopFuture<[Result]> where Result: Decodable {
```

Comment:
I think this has use outside of `isUnique` being true, too. If we remove the label and don't set `isUnique`, then this can be used like:
```swift
Planet.query(on: db).all(\.$name)
// or
Planet.query(on: db).unique().all(\.$name)
```

Reply:
`all(selecting: \.$name)`? Agree it can be more generic. Will think about a better name